Adding `meta_desc` to blogs (first batch) (#1499)
pulumi/docs: content/blog/deploying-production-ready-containers-with-pulumi/index.md

title: "Deploying production-ready containers with Pulumi"
authors: ["donna-malayeri"]
tags: ["JavaScript","AWS","Containers"]

> meta_desc: "Deploy a simple Docker container running NGINX.Pulumi makes it easy to deploy production Docker containers, handling details such as creating a container registry instance in ECR, creating task definitions in ECS, and configuring a load balancer. "

Reviewer: ```suggestion
meta_desc: "Deploy a simple Docker container running NGINX.Pulumi showing details such as creating a container registry instance in ECR, creating task definitions in ECS, and configuring a load balancer. "
```
